apps/site/package.json

```diff
@@ -42,7 +42,8 @@
     "@twind/next": "^1.0.9",
     "ajv": "^8.9.0",
     "algoliasearch": "4.12.1",
-    "axios": "^1.1.3",
+    "axios": "1.2.0-alpha.1",
```

Comment:
There is a compatibility bug in axios 1.x. I could have downgraded to 0.27 (https://github.com/axios/axios/issues/5163#issuecomment-1301549669), but I've decided to use a newer alpha instead (https://github.com/axios/axios/issues/5040#issuecomment-1310791676).